dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/TrackerImportParams.java

```diff
@@ -43,9 +43,6 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class TrackerImportParams implements JobParameters {
-  /** User uid to use for import job. */
-  @JsonProperty private String userId;
```

Comment:
JobParameters are stored as JSONB in the DB. They usually need a migration. Does this change not need one?

Reply:
I didn't think about it but probably this is ok.
This `userId` was set when mapping `ImportRequestParams` to `TrackerImportParams` and the first one didn't have a `userId`.
I am going to double check how the `JobParameters` object is created and if this change can cause any issue.